client/src/features/catalog/Catalog.tsx

```diff
@@ -12,8 +14,19 @@ export default function Catalog() {
     }, [])
 
     return (
-        <>
-            <ProductList products={products} />
-        </>
+        <div>
```

Comment:
do you need this `div`? The less code the better so here the less markup the better. if this div is not really needed then I propose to replace it with <></>